tests/Model/Generated/Endpoint/PaymentBatchTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace bunq\test\Model\Generated\Endpoint;
+
+use bunq\Model\Generated\Endpoint\Payment;
+use bunq\Model\Generated\Endpoint\PaymentBatch;
+use bunq\Model\Generated\Object\Amount;
+use bunq\Model\Generated\Object\LabelMonetaryAccount;
+use bunq\Model\Generated\Object\Pointer;
+use bunq\test\BunqSdkTestBase;
+
+/**
```

Comment:
This is so ugly. Why don't we leave it out when we don't adhere strictly to internal codestyle guidelines anyway?